cdap-watchdog-api/src/main/java/co/cask/cdap/api/metrics/MetricDataQuery.java

```diff
@@ -41,8 +41,20 @@
  * </pre>
  */
 public final class MetricDataQuery {
+
+  /**
+   * Start timestamp, in seconds.
+   */
   private final long startTs;
+
+  /**
+   * End timestamp, in seconds.
+   */
   private final long endTs;
+
```

Comment:
Thanks for adding that. Javadocs has been missing from this class, and you probably tried to use millis first? 
While you're at it: It would make sense to add this information to the constructor, too.